### PR TITLE
fix bug in the transformation from to_variable to assign

### PR DESCRIPTION
**Pull request by liym27**
1. Support the input of `fluid.dygraph.to_variable` is a numpy statement in the transformation of dygraph to static graph;
2. Align parameters between api `to_variable` and `assign`.

### CODE REVIEW

**Review by zhhsplendid — APPROVED** [review]
LGTM

File: python/paddle/fluid/dygraph/dygraph_to_static/ast_utils.py [hunk 1]
```diff
@@ -360,7 +360,9 @@ def ast_to_func(ast_root, func_name, delete_on_exit=True):
     # TODO(Aurelius84): more elegant way to transform ast into callable object
     import_str = "import paddle\n" \
                  "import paddle.fluid as fluid\n" \
-                 "import paddle.fluid.layers as layers\n"
+                 "import paddle.fluid.layers as layers\n" \
+                 "import numpy as np\n" \
+                 "import numpy\n"
```
Comment (Aurelius84) [inline python/paddle/fluid/dygraph/dygraph_to_static/ast_utils.py:365]: Here may lead to conflicts with [PR 22830](https://github.com/PaddlePaddle/Paddle/pull/22830/files)

File: python/paddle/fluid/tests/unittests/test_dygraph_to_static_basic_api_transformation.py [hunk 2]
```diff
@@ -28,19 +28,25 @@
 
 
 def dyfunc_to_variable(x):
-    res = fluid.dygraph.to_variable(x)
+    res = fluid.dygraph.to_variable(x, name=None, zero_copy=None)
+    return res
```
Comment (Aurelius84) [inline python/paddle/fluid/tests/unittests/test_dygraph_to_static_basic_api_transformation.py:32]: If you should push a commit to fix conflict, could you move this unittest file to dir `dygraph_to_static`?

Comment (liym27) [reply python/paddle/fluid/tests/unittests/test_dygraph_to_static_basic_api_transformation.py:32]: OK. And other test files about dygraph_to_static will be moved.

**Pull request merged**